pkg/workflow/types.go

```diff
@@ -35,6 +35,8 @@ type Data interface {
 	SetContentLocation(string)
 	AddError(err snyk_errors.Error)
 	GetErrorList() []snyk_errors.Error
+	SetConfiguration(config configuration.Configuration)
```

Comment:
Question: Do we really need this here? otherwise I would say configuration is to big of a concept to link it to on data instance. better take what you need from the configuration once.

Reply:
removed